management_interface/management_interface/utils.py

```diff
@@ -0,0 +1,7 @@
+from django.core.exceptions import ValidationError
+
+
+def validated_email_domain(email: str, domain: str, error_message: str):
```

Comment:
See above - I don't think we need this if we have `SecureEmailValidator`.

Reply:
Removed.